data/css/_project-setup.scss

```diff
@@ -6,6 +6,3 @@
 
 $images-path: 'resource:///com/endlessm/knowledge/data/images/';
```

Comment:
I wonder if we can ditch this project setup entirely with the switch away from compass, I suspect we can.